app/views/pages/contact.ctp

```diff
@@ -93,12 +93,16 @@ $this->set('title_for_layout', $pages->formatTitle(__('Contact us', true)));
     
     
     <div class="module">
-        <h2><?php __('Join us on IRC'); ?></h2>
+        <h2><?php __('Join us on Jabber'); ?></h2>
+        <p>
         <?php
         echo format(
             __(
-                'We also have an IRC channel on freenode, #tatoeba. If you are not '.
-                'familiar with IRC, you can read the <a href="{}">Help</a>.', true
+                'We also have a Jabber room, <a '.
+                'href="xmpp:tatoeba@chat.tatoeba.org?join">'.
+                'tatoeba@chat.tatoeba.org</a>, also available with the web'.
+                'client below.  If you are not familiar with Jabber, you '.
+                'can read the <a href="{}">Help</a>.', true
```

Comment:
You should put the href of the first link out of the string so that it’s easier to translate. See [this example](https://github.com/Tatoeba/tatoeba2/blob/39fbbb5301ce10423dbd591d5c08e9382ce859d0/app/views/pages/donate.ctp#L36-L44).